packages/js/uri-resolvers/src/helpers/RetryResolver.ts

```diff
@@ -0,0 +1,113 @@
+import { UriResolverLike } from "./UriResolverLike";
+import { UriResolver } from "./UriResolver";
+import { UriResolutionResult } from "./UriResolutionResult";
+
+import {
+  CoreClient,
+  IUriResolutionContext,
+  IUriResolver,
+  Result,
+  Uri,
+  UriPackageOrWrapper,
+} from "@polywrap/core-js";
+
+/** configure how additional resolution attempts are handled after an initial resolution attempt fails */
+export type RetryOptions = {
+  /** the number of additional resolution attempts */
+  retries: number;
+  /** the duration (in ms) to pause between attempts */
+  interval: number;
+};
+
+/** A map or uri or authority strings to retry options */
```

Comment:
Typo: `A map of URI or authority...`
`of` not `or`

Reply:
no, it's `or`

They key can be a URI or a URI authority

Reply:
Nice catch